rosdep/base.yaml

```
@@ -1929,7 +1929,9 @@ libav:
   debian: [libav-tools]
   fedora: []
   gentoo: [virtual/ffmpeg]
-  ubuntu: [libav-tools]
+  ubuntu:
+    '*': [ffmpeg]
+    xenial: [libav-tools]
```

I understand where you are coming from, but as far as I can tell `ffmpeg` and `libav-tools` don't provide the same resources.  That is, on Xenial it looks like `libav-tools` provides the binaries `avconv`, `avprobe`, and `avplay`, while the `ffmpeg` package on newer distributions provides the binaries `ffmpeg`, `ffplay`, and `ffprobe`.  While those are the same in spirit, anything depending on being able to call `avplay` is going to fail.

What I'd suggest here is nulling out this entry for anything newer than Xenial.  That way packages will have to switch their package.xml to the `ffmpeg` key and update their code at the same time.

```suggestion
  ubuntu:
    '*': null
    xenial: [libav-tools]
```

---

You're right, null entry is better describing the reality.